src/components/Table.vue

```diff
@@ -341,6 +342,7 @@ export default {
     responsive: { default: true },
     rtl: { default: false },
     rowStyleClass: { default: null, type: [Function, String] },
+    compactMode: { default: false, type: Boolean},
```

Comment:
Properties that you pass as an attribute that are boolean and default false, can just be specified like: `compactMode: Boolean`. Then you can pass it like an attribute without value. Like: `<component compact-mode />`

But I'm not the dev. 
Very nice feature though!

Reply:
sure. thanks for feedback

Reply:
added your suggestion.